flake.nix

```diff
@@ -77,18 +77,27 @@
             enableDebug = false;
             crypto3 = crypto3;
             evm-assigner = evm-assigner;
+            transpiler = transpiler;
+            proof-producer = proof-producer;
+            parallel-crypto3 = parallel-crypto3;
```

Comment:
not exactly sure what do you mean.
Set only proof-producer - not enough, `transpiler` and `parallel-crypto3` should be provided as well

Reply:
Sorry, I was confused by `callPackage` definition. Indeed, these arguments are necessary. Let's replace then with `inherit transpiler proof-producer parallel-crypto3;` It's basically the same but more common in Nix.

Reply:
for use `inherit` we'll have to define `transpiler proof-producer parallel-crypto3` before `packages` and use `inherit transpiler proof-producer parallel-crypto3` or all `zkevm-ramework*`
It will violate common style, so need to do same for all another packages. I guess this common changes should be in the separate PR

Reply:
Didn't get why we would need to change definition order, `inherit` is just another notation of the same expression. But I agree that this could be done along with refactoring of other packages in separate PR.